### ISSUE TITLE
valid and test datasets of wikipedia are missing

### ISSUE THREAD

**Issue by Xuer0817** [AaronHeee/Reindex-Then-Adapt#1]
Hello authors, under https://huggingface.co/datasets/ZhankuiHe/Reindex-Then-Adapt-LLM-Generated-Data/tree/main/wikipedia, there is only a 'train' folder, and the 'valid' and 'test' folders are missing. Could you please upload them? Thank you!

**Comment by AaronHeee** [comment]
Hi @Xuer0817, thanks for asking! In the experiments, we mixed Wikipedia data with other CRS data and selected the best hyper-parameters based on the validation set of CRS data. So Wikipedia is not used in validation or testing, thus those folders are empty. Our code handles it specially.
